dist_tables/append_distribution.rst

```diff
@@ -55,38 +55,41 @@ This function informs Citus that the github_events table should be distributed b
 Data Loading
 ------------
 
-Citus supports two methods to load data into your append distributed tables. The first one is suitable for bulk loads from CSV/TSV files and involves using the \stage command. For use cases requiring smaller, incremental data loads, Citus provides two user defined functions. We describe each of the methods and their usage below.
+Citus supports two methods to load data into your append distributed tables. The first one is suitable for bulk loads from CSV/TSV files and involves using the \copy command. For use cases requiring smaller, incremental data loads, Citus provides two user defined functions. We describe each of the methods and their usage below.
 
-Bulk load using \\stage
+Bulk load using \\copy
 $$$$$$$$$$$$$$$$$$$$$$$
 
-The \stage command is used to copy data from a file to a distributed table while handling replication and failures automatically. It is not available in the standard `psql <http://www.postgresql.org/docs/9.5/static/app-psql.html>`_ client. Instead, Citus provides a database client called csql which supports all the commands of psql plus the \stage command.
+The `\\copy <http://www.postgresql.org/docs/current/static/app-psql.html#APP-PSQL-META-COMMANDS-COPY>`_
+command is used to copy data from a file to a distributed table while handling
+replication and failures automatically. You can also use the server side `COPY command <http://www.postgresql.org/docs/current/static/sql-copy.html>`_. 
+However, in general, \\copy is preferable over COPY in examples, since the latter requires superuser, whereas \\copy does not. \\copy actually sends a COPY .. FROM STDIN to the server.
```

Comment:
Maybe instead of this sentence "In the examples, we use the \copy command from psql, which sends a COPY .. FROM STDIN to the server and reads files on the client side, whereas COPY would read them from the server and that requires superuser. "